tests/tracking/test_tracking.py

```diff
@@ -225,9 +247,8 @@ def get_store_mock(tmpdir):
 def test_set_tags(tracking_uri_mock):
     exact_expected_tags = {"name_1": "c", "name_2": "b", "nested/nested/name": "5"}
     approx_expected_tags = set([MLFLOW_SOURCE_NAME, MLFLOW_SOURCE_TYPE])
-    active_run = start_run()
-    run_uuid = active_run.info.run_uuid
-    with active_run:
+    with start_run() as active_run:
```

Comment:
Note: all changes from here onwards are refactoring related to wrapping the return value of `start_run` in a `with` block